docs_outdoornav_user_manual/web_user_interface/ui_overview.mdx

```diff
@@ -226,6 +228,43 @@ acquired by the VLP-16 LiDAR.
 
 ## System Configuration
 
+### General Settings
+
+The General settings section can be found in through the upper left hand menu and allows the Operator to modify 
+general features of the UGV.
+
+<center>
+  <figure>
+    <img src="/img/outdoornav_images/ui_general_settings.png" />
+    <figcaption>General Settings Page</figcaption>
+  </figure>
+</center>
+
+#### Coordinates
+
+The Operator can change the coordinate space from X/Y relative to the Datum to Latitude/Longitude.
+
+#### Save Image Location 
+
+The Operator can choose to store images saved manually to the robot directly or to download them onto their computer. This 
+feature only affects the manual save image option found on each relevant camera view.
+
+#### Robot Internet Connection Type 
+
+If the UGV is equipped with SIM card and can switch between Cellular and WiFi connections, the Operator can manually trigger this 
+change through the general settings page.  This switchover will take approximately 30 seconds and will require a refresh on the UI.
+
+#### Show Path Deviation Distance
+
+When a mission is running the UGV will have a maximum path deviation distance that it shouldn't cross if it needs to replan.  If this 
+toggle is set to true, while in edit mode this buffer is shown over top of the path as the Operator creates/edits the mission.  This 
+toggle does not disable the path deviation distance feature.
+
+#### Low Power Mode
+
+If the UGV is equipped with a Low Power Module this toggle allows the user to switch the UGV into a low power state that will drastically 
+limit functionality but help conserve power.

```

Comment:
Perhaps add a note that the low power mode is most applicable during charging while the system is not being used.